src/ILCompiler.ReadyToRun/src/Compiler/DependencyAnalysis/ReadyToRun/GCRefMapBuilder.cs

```diff
@@ -100,7 +100,7 @@ public void GetCallRefMap(MethodDesc method)
             {
                 // Flush an empty GC ref map block so that the list of methods remains in sync with the list of GC ref map records
                 Flush();
-                return;
+                return false;
```

Comment:
Why not throw from here right away? We should not be ever building the RefMap if the method has arguments of unknown sizes.

Reply:
Good suggestion, thanks; applied in 3rd commit.